artifacts/rkbin/rk3588/README.md

```diff
@@ -0,0 +1,26 @@
+# README for DDR Binary Patching
```

Comment:
this actually can be a different stage that uses debian as base and copy over the patched file from that stage (I assume the patched data is reproducible)

Reply:
Yes, if I now how to use debian in BLDR in a stage? I can't find examples

Reply:
actually use wolfi:base https://github.com/siderolabs/extensions/blob/main/nvidia-gpu/nvidia-container-toolkit/nvidia-container-cli/elfutils/pkg.yaml#L1-L9 as this, it's glibc based and small

Reply:
done